palette/moreland/smooth_test.go

```diff
@@ -246,3 +246,11 @@ func similar(a, b color.Color, tolerance float64) bool {
 	}
 	return true
 }
+
+func Test_smoothDiverging_Palette(t *testing.T) {
```

Comment:
```suggestion
func TestIssue798SmoothBlueRed(t *testing.T) {
```

Reply:
Updated in 55662b36b4613f9934453412090318e259a40308.